armi/materials/material.py

```diff
@@ -214,19 +215,19 @@ def linearExpansionFactor(self, Tc: float, T0: float) -> float:
         --------
         linearExpansionPercent
         """
-        dLLhot = self.linearExpansionPercent(Tc=Tc)
-        dLLcold = self.linearExpansionPercent(Tc=T0)
+        dLLhot = self.linearExpansionPercent(Tc=Tc) / 100.0
+        dLLcold = self.linearExpansionPercent(Tc=T0) / 100.0
 
-        return (dLLhot - dLLcold) / (100.0 + dLLcold)
+        return exp(dLLhot) / exp(dLLcold)
```

Comment:
It probably is faster to just do `exp(dLLhot - dLLcold)`

Reply:
Addressed https://github.com/terrapower/armi/pull/1279/commits/014a53503c227f55ade39d92cc818e3d3e381f34